examples/migrate_cluster_to_advanced_cluster/module_user/README.md

```diff
@@ -0,0 +1,104 @@
+# Module User - `mongodbatlas_cluster` to `mongodbatlas_advanced_cluster`
+
+The purpose of this example is to demonstrate the experience of adopting a new version of a terraform module definition which internally migrated from `mongodbatlas_cluster` to `mongodbatlas_advanced_cluster`.
+Each module call represent a step on the migration path.
+The example focus on the call of the module rather than the module implementation itself (see the [module maintainer README.md](../module_maintainer/README.md) for the implementation details)
```

Comment:
```suggestion
The example focus on the call of the module rather than the module implementation itself (see the [module maintainer README.md](../module_maintainer/README.md) for the implementation details).
```